src/test/java/BMICalculator.java

```diff
@@ -3,24 +3,25 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import java.util.ArrayList;
 import java.util.Set;
 
 public class BMICalculator {
     String address = "https://healthunify.com/bmicalculator/";
+    WebDriver driver = new ChromeDriver();
 
-    public void sys() {
-        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
 
+    @BeforeMethod
+    public void initMethod() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
     }
 
     @Test
     //This test should verify Normal category on home page
     public void calculateNormal() {
-        sys();
-        WebDriver driver = new ChromeDriver();
```

Comment:
You already have test code in master. In master should be only pom.xml file in initial commit. All test code should be in feature branch.